frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -33,6 +35,36 @@
 # pylint: enable=import-error,no-name-in-module
 
 
+# replace missing tensor.FromElementsOp.__init__
+class _FromElementsOp(tensor.FromElementsOp):
```

Comment:
Let's not define this class here, maybe on utils.py?

Reply:
Done